### ISSUE TITLE
Plugin autostart not working 

### ISSUE THREAD

**Issue by ccruson** [ONLYOFFICE/DocumentServer#241]
Pretty Sure this is a bug.

Followed directions here: https://api.onlyoffice.com/editors/config/editor/plugins.
when I load the editor it glitches rendering only an empty screen. 

This is what the error log reports: 
```
app.js:53 Uncaught TypeError: Cannot read property 'get_Guid' of null
    at V.i.onPluginClose (app.js:53)
    at M (app.js:8)
    at y.<anonymous> (app.js:8)
    at y.<anonymous> (app.js:8)
    at y.yd (sdk-all-min.js:623)
    at close (sdk-all.js:9563)
    at c.wWb (sdk-all.js:9565)
    at y.Ja.C0d (sdk-all-min.js:561)
    at V.i.onPluginShow (app.js:53)
    at M (app.js:8)
    at y.<anonymous> (app.js:8)
    at y.<anonymous> (app.js:8)
    at y.yd (sdk-all-min.js:623)
    at c.show (sdk-all.js:9564)
    at c.Yy (sdk-all.js:9563)
    at y.Ja.o7a (sdk-all-min.js:561)
    at V.i.runAutoStartPlugins (app.js:53)
    at V.i.updatePluginsList (app.js:50)
    at V.i.requestPlugins (app.js:50)
    at V.i.onDocumentContentReady (app.js:49)
    at M (app.js:8)
    at y.<anonymous> (app.js:8)
    at y.<anonymous> (app.js:8)
    at y.yd (sdk-all-min.js:623)
    at y.y7a (sdk-all-min.js:727)
    at Object.Yaa (sdk-all.js:5380)
    at y.CIc (sdk-all-min.js:724)
    at Object.l6a (sdk-all.js:5378)
    at Object.l6a (sdk-all.js:5378)
    at Object.l6a (sdk-all.js:5378)
    at Object.l6a (sdk-all.js:5378)
    at Object.l6a (sdk-all.js:5378)
    at Object.gMb (sdk-all.js:5377)
    at y.jXc (sdk-all-min.js:731)
    at fx.Xa.$od (sdk-all-min.js:501)
    at fx.Xa.B0a (sdk-all-min.js:500)
    at y.nlb (sdk-all-min.js:728)
    at y.y7a (sdk-all-min.js:727)
    at Object.Yaa (sdk-all.js:5380)
    at y.CIc (sdk-all-min.js:725)
    at Object.l6a (sdk-all.js:5378)
    at cFc (sdk-all.js:5379)
onPluginClose @ app.js:53
M @ app.js:8
(anonymous) @ app.js:8
(anonymous) @ app.js:8
y.yd @ sdk-all-min.js:623
close @ sdk-all.js:9563
wWb @ sdk-all.js:9565
Ja.C0d @ sdk-all-min.js:561
onPluginShow @ app.js:53
M @ app.js:8
(anonymous) @ app.js:8
(anonymous) @ app.js:8
y.yd @ sdk-all-min.js:623
show @ sdk-all.js:9564
Yy @ sdk-all.js:9563
Ja.o7a @ sdk-all-min.js:561
runAutoStartPlugins @ app.js:53
updatePluginsList @ app.js:50
requestPlugins @ app.js:50
onDocumentContentReady @ app.js:49
M @ app.js:8
(anonymous) @ app.js:8
(anonymous) @ app.js:8
y.yd @ sdk-all-min.js:623
y.y7a @ sdk-all-min.js:727
Yaa @ sdk-all.js:5380
y.CIc @ sdk-all-min.js:724
l6a @ sdk-all.js:5378
l6a @ sdk-all.js:5378
l6a @ sdk-all.js:5378
l6a @ sdk-all.js:5378
l6a @ sdk-all.js:5378
gMb @ sdk-all.js:5377
y.jXc @ sdk-all-min.js:731
Xa.$od @ sdk-all-min.js:501
Xa.B0a @ sdk-all-min.js:500
y.nlb @ sdk-all-min.js:728
y.y7a @ sdk-all-min.js:727
Yaa @ sdk-all.js:5380
y.CIc @ sdk-all-min.js:725
l6a @ sdk-all.js:5378
cFc @ sdk-all.js:5379
setTimeout (async)
l6a @ sdk-all.js:5378
l6a @ sdk-all.js:5378
l6a @ sdk-all.js:5378
l6a @ sdk-all.js:5378
l6a @ sdk-all.js:5378
cFc @ sdk-all.js:5379
setTimeout (async)
l6a @ sdk-all.js:5378
l6a @ sdk-all.js:5378
cFc @ sdk-all.js:5379
setTimeout (async)
l6a @ sdk-all.js:5378
cFc @ sdk-all.js:5379
setTimeout (async)
cFc @ sdk-all.js:5378
setTimeout (async)
cFc @ sdk-all.js:5378
setTimeout (async)
cFc @ sdk-all.js:5378
setTimeout (async)
l6a @ sdk-all.js:5378
sDa @ sdk-all.js:5377
y.mlb @ sdk-all-min.js:635
y.PYa @ sdk-all-min.js:730
Ja.xWc @ sdk-all-min.js:556
(anonymous) @ sdk-all-min.js:741
tb @ sdk-all-min.js:321
(anonymous) @ sdk-all-min.js:321
Lf.Hb.onreadystatechange @ sdk-all-min.js:290
XMLHttpRequest.send (async)
(anonymous) @ sdk-all-min.js:290
Lf @ sdk-all-min.js:290
Db @ sdk-all-min.js:279
f.AscCommon.Foe @ sdk-all-min.js:321
y.LSb @ sdk-all-min.js:740
Ck.Sza @ sdk-all-min.js:549
Ja.a6d @ sdk-all-min.js:99
Ij.Sza @ sdk-all-min.js:92
Na.xRd @ sdk-all-min.js:111
Na.XSd @ sdk-all-min.js:122
y.onmessage @ sdk-all-min.js:121
r.dispatchEvent @ sockjs.min.js:2
(anonymous) @ sockjs.min.js:2
r._transportMessage @ sockjs.min.js:2
r.emit @ sockjs.min.js:2
ws.onmessage @ sockjs.min.js:3
```

Went deep into the app.js and found that `t.get_Guid` doesn't work because t is undefined in the function `onPluginClose` when it is bound to `asc_onPluginClose` in the `setAPI` method.

This is happening with Document Server 5.0.99.117
with a doc editor config like this:
```
{  
   "width":"100%",
   "height":"100%",
   "type":"desktop",
   "documentType":"text",
   "token":"",
   "document":{  
      "title":"texas_58b0f2fc964fd8404b51df78_feb 2_5a74f1cce82c1a22c2ff3310.docx",
      "url":"http://174.138.56.53:5000/files//texas_58b0f2fc964fd8404b51df78_feb%202_5a74f1cce82c1a22c2ff3310.docx",
      "fileType":"docx",
      "key":"1964486775",
      "info":{  
         "author":"Me",
         "created":"Fri Feb 16 2018"
      },
      "permissions":{  
         "download":true,
         "edit":"true",
         "review":true
      }
   },
   "editorConfig":{  
      "mode":"edit",
      "lang":"en",
      "callbackUrl":"http://174.138.56.53:5000/track?filename=texas_58b0f2fc964fd8404b51df78_feb%202_5a74f1cce82c1a22c2ff3310.docx&useraddress=__ffff_136.62.90.227",
      "user":{  
         "id":"58b0f2fc964fd8404b51df78",
         "name":"Dummy One"
      },
      "embedded":{  
         "saveUrl":"http://174.138.56.53:5000/files//texas_58b0f2fc964fd8404b51df78_feb%202_5a74f1cce82c1a22c2ff3310.docx",
         "embedUrl":"http://174.138.56.53:5000/files//texas_58b0f2fc964fd8404b51df78_feb%202_5a74f1cce82c1a22c2ff3310.docx",
         "shareUrl":"http://174.138.56.53:5000/files//texas_58b0f2fc964fd8404b51df78_feb%202_5a74f1cce82c1a22c2ff3310.docx",
         "toolbarDocked":"top"
      },
      "customization":{  
         "about":true,
         "chat":true,
         "comments":true,
         "feedback":true,
         "forcesave":false,
         "goback":{  
            "url":"http://174.138.56.53:5000"
         }
      },
      "fileChoiceUrl":"",
      "plugins":{  
         "autostart":[  
            "asc.{C05D81BC-92C7-4C48-BA49-E8B9AAFCD776}"
         ],
         "pluginsData":[  
            "http://165.227.78.201/2017-10-06-15-29/sdkjs-plugins/example_legaldocs/config.json"
         ]
      }
   },
   "events":{  
      "onAppReady":onAppReady,
      "onDocumentStateChange":onDocumentStateChange,
      "onRequestEditRights":onRequestEditRights,
      "onError":onError,
      "onRequestHistory":onRequestHistory,
      "onRequestHistoryData":onRequestHistoryData,
      "onRequestHistoryClose":onRequestHistoryClose,
      "onOutdatedVersion":onOutdatedVersion,

   }
}
```
and a plugin config lie this: 
```
{
    "baseUrl": "",
    "guid" : "asc.{C05D81BC-92C7-4C48-BA49-E8B9AAFCD776}",
    "name" : "LegalDocs plugin",
    "variations" : [
        {
            "description" : "LegalDocs plugin",
            "url"         : "index.html",
            "icons"           : ["icon.png", "icon@2x.png", "icon2.png", "icon2@2x.png"],
            "isViewer"        : false,
            "EditorsSupport"  : ["word"],
            "isVisual"        : true,
            "isModal"         : false,
            "isInsideMode"    : true,
            "initDataType"    : "none",
            "initData"        : "",
            "isUpdateOleOnResize" : false,
            "initOnSelectionChanged": false,
            "isInsideMode": true,
            "isViewer": false,
            "buttons"         : []
        }
    ]
}
```

**What is the expected behavior?**
That the plugin load and open when the document is finished loading

**Did this work in previous versions of DocumentServer?**
I don't know, this is the first time I've tried. https://api.onlyoffice.com/editors/changelog indicates it should work with version 5

**DocumentServer version:**
5.0.99.117

**Operating System:**
The Document server is the docker image: https://hub.docker.com/r/onlyoffice/4testing-documentserver-integration/

**Browser version:**
Using Chromium Version 64.0.3282.140

**Comment by ShockwaveNN** [comment]
> DocumentServer version:
> 5.0.99.117

This is pretty old dev version of DocumentServer, currently 5.0.99.522
And BTW we do not guarantee working of `4testing` images in any way
Is this issue actual on latest version?

**Comment by ccruson** [comment]
okay so updated the version to the one you listed above. doesn't throw an error. But also doesn't autostart the plugin. Any ideas? is my config off in some way?

**Comment by maxkadushkin** [comment]
your config is correct. in that version there is an issue with plugins designed for 'left' plugins panel
"isInsideMode": true
bug doesn't affect other types of plugins.  it'll fixed in next release

**Comment by ShockwaveNN** [comment]
@ccruson your problem is fixed in https://github.com/ONLYOFFICE/web-apps-pro/commit/d08137da83ebb9f9fc1411e6a073f821552f0ab9
I checked `onlyoffice/4testing-documentserver-ie:5.0.99.543` with default Clipart plugin and config with this lines and Clipart is correctly shown on start:
```
"plugins":{  
         "autostart":[  
            "asc.{F5BACB61-64C5-4711-AC8A-D01EC3B2B6F1}"
         ],
         "pluginsData":[  
            "http://localhost/2018-02-15-17-16/sdkjs-plugins/clipart/config.json"
         ]
      }
```

This fix will be in stable release of 5.1

**Comment by ccruson** [comment]
Thanks
